crates/swc_ecma_minifier/tests/fixture/issues/6463/output.js

```diff
@@ -0,0 +1,4 @@
+var foo_1 = function() {
```

Comment:
Is it not possible to avoid the reassignment?

Reply:
I think it's possible. I'll check, thank you!

Reply:
I tried, but we can't detect it while deciding to inline a function declaration. We need a costly reverse map to access `foo` from `bar = foo`.

I'll revisit this in the future